### PR TITLE
Multi display in menu bar (toolbar)

### PR DESCRIPTION
**Pull request by NicKoehler**
Implemented what discussed in #3333

| Enabled | Disabled |
| --- | --- |
| image | image |
| image | image |

### CODE REVIEW

File: flutter/lib/desktop/widgets/remote_menubar.dart [hunk 1]
```diff
@@ -391,7 +391,16 @@ class _RemoteMenubarState extends State<RemoteMenubar> {
           _FullscreenMenu(state: widget.state, setFullscreen: _setFullscreen));
       menubarItems.add(_MobileActionMenu(ffi: widget.ffi));
     }
-    menubarItems.add(_MonitorMenu(id: widget.id, ffi: widget.ffi));
+
+    if (PrivacyModeState.find(widget.id).isTrue ||
+        stateGlobal.displaysCount.value > 1) {
```
Comment (rustdesk) [inline flutter/lib/desktop/widgets/remote_menubar.dart:397]: Logic is wrong here.

```
!(PrivacyModeState.find(id).isTrue ||
        stateGlobal.displaysCount.value < 2) 
```
=>

```
PrivacyModeState.find(id).isFalse && stateGlobal.displaysCount.value > 1
```

**Comment by rustdesk** [comment]
Sorry, there are conflicts.

**Comment by rustdesk** [comment]
Could you paste current snapshot?

**Comment by rustdesk** [comment]
Can we change the key of `Show monitors in menu bar` to `show_monitors_tip`? then it will be easier to modify its content in the future.

**Comment by rustdesk** [comment]
BTW @grummbeer @NicKoehler should this be called menu bar or toolbar?

**Comment by NicKoehler** [comment]
> Could you paste current snapshot for two cases?

I updated the pr description.


> BTW @grummbeer @NicKoehler should this be called menu bar or toolbar?

Is called MenuBar everywhere (filename, classes), but I think toolbar would be the proper name

**Comment by grummbeer** [comment]
Its a toolbar.

**Comment by rustdesk** [comment]
> Is called MenuBar everywhere

That's my fault, did not review their code seriously before.

File: flutter/lib/desktop/widgets/remote_menubar.dart (hunk 1 above)
Comment (rustdesk) [reply flutter/lib/desktop/widgets/remote_menubar.dart:397]: @NicKoehler please fix this.

Comment (NicKoehler) [reply flutter/lib/desktop/widgets/remote_menubar.dart:397]: fixed

**Comment by grummbeer** [comment]
:+1:

**Comment by ForsakenRei** [comment]
Thank you!

**Pull request merged**